README.md

```diff
@@ -169,6 +169,23 @@ list_files = drive.list_files()
 
 ```
 
+### Example (Oauth 2.0 Web Server application)
+
+```ruby
+  require 'google/api_client/client_secrets.rb'
+  require "google/apis/oauth2_v2"
+
+  Google::Apis::Oauth2V2::Oauth2Service.new.tap do |client|
+    authorization = Google::APIClient::ClientSecrets.new(
```

Comment:
How to install the client secret package with their new generated package system?
"google-api-client" is deprecated.

Reply:
It's been moved to google-apis-core.